private-templates-service/adaptivecards-templating-service/src/models/mongo/UserModel.ts

```diff
@@ -0,0 +1,18 @@
+import mongoose, { Schema } from "mongoose";
+import { IUser } from "../models";
+
+export interface IUserModel extends mongoose.Document, IUser {
```

Comment:
Why not 'implement' the IUser interface, rather than extending it?

Reply:
one interface cannot implement another, it has to extend it